src/parser/jobs/ast/modules/oGCDs.tsx

```diff
@@ -0,0 +1,125 @@
+import {Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {dependency} from 'parser/core/Injectable'
+import {CooldownDowntime} from 'parser/core/modules/CooldownDowntime'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React, {Fragment} from 'react'
+
+//assumptions listed after each severity
+const SEVERITY_PERCENTAGES = {
+	HEALING_PERCENT_THRESHOLD: {
+		0.8: SEVERITY.MAJOR, //less than 20% of the available time is close to not using it at all or barely //based on 80/20
+		0.4: SEVERITY.MEDIUM, //60% is not using it enough -- risks having to use GCDs to make up for lack of mitigation/heals
+		0.2: SEVERITY.MINOR, //80% of the time is used to keep it on the radar, but not punish //based on 80/20
+	},
+	DPS_PERCENT_THRESHOLD: {
+		0.25: SEVERITY.MAJOR,
+		0.15: SEVERITY.MEDIUM,
+		0.05: SEVERITY.MINOR,
+	},
+}
+
+//assumptions on importance of healing and dps by weight. //major assumption is that dps metrics are more important than healing metrics as long as no one dies to use these dps buffs.
+const WEIGHTS = {
+	HEALS: 0.25,
+	DPS: 1,
+}
+
+//type for action
+const TYPES = {
+	HEALS: 'heals',
+	DPS: 'dps',
+}
+
+export default class oGCDs extends CooldownDowntime {
```

Comment:
No default export
```suggestion
export class oGCDs extends CooldownDowntime {
```

Reply:
I get this error when I do that
![image](https://user-images.githubusercontent.com/89435408/145928455-dac5cda4-83aa-4ddb-a20f-aca9af3c1ff2.png)

Reply:
works beautifully
thanks!